### ISSUE TITLE
can't keyboard navigate through the About dialogue back to the close button

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/friction#78]
<b>Test Device</b>

MacBook Air

<b>Operating System</b>

macOS 10.13.4

<b>Browser</b>

Safari 11.1, Firefox 60.0.1, Chrome 66.0.3359.181

<b>Problem Description</b>

For phetsims/QA/issues/127. I can't tab through the About dialogue.

<b>Steps to Reproduce</b>

1. Keyboard navigate to the PhET menu.
2. Keyboard navigate to the About dialogue.
3. Try to tab through the About dialogue.

<b>Visuals</b>

Incorrect Behavior:
![friction](https://user-images.githubusercontent.com/31285590/41062796-1371c6c2-6994-11e8-8624-6adb106b70f9.gif)

Correct Behavior:
![john](https://user-images.githubusercontent.com/31285590/41063366-aa03a460-6995-11e8-9b75-e7c8d5d8eb41.gif)

<details>
<summary><b>Troubleshooting Information</b></summary>

Name: ‪John Travoltage‬
URL: https://phet.colorado.edu/sims/html/john-travoltage/latest/john-travoltage_en.html
Version: 1.3.2 2018-04-27 21:07:16 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36
Language: en-US
Window: 1920x983
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"5c3f9204","branch":"HEAD"},"axon":{"sha":"7b0852e8","branch":"HEAD"},"babel":{"sha":"48e135de","branch":"master"},"brand":{"sha":"8e878774","branch":"HEAD"},"chipper":{"sha":"3c76a864","branch":"HEAD"},"dot":{"sha":"6b4ec39c","branch":"HEAD"},"john-travoltage":{"sha":"27358b7f","branch":"HEAD"},"joist":{"sha":"63ef27ce","branch":"HEAD"},"kite":{"sha":"7d2df183","branch":"HEAD"},"phet-core":{"sha":"76ec0204","branch":"HEAD"},"phetcommon":{"sha":"11406a81","branch":"HEAD"},"query-string-machine":{"sha":"c74e454e","branch":"HEAD"},"scenery":{"sha":"d6730968","branch":"HEAD"},"scenery-phet":{"sha":"78a67d1d","branch":"HEAD"},"sherpa":{"sha":"0e326d54","branch":"HEAD"},"sun":{"sha":"1c94ce39","branch":"HEAD"},"tandem":{"sha":"744bc32f","branch":"HEAD"},"vibe":{"sha":"5e8151e1","branch":"HEAD"}}

</details>

**Comment by jessegreenberg** [comment]
I am not able to reproduce in Chrome on Windows in master or in https://bayes.colorado.edu/dev/html/friction/1.5.0-dev.6/phet/friction_all_phet.html. This is only happening in macOS.

**Comment by jessegreenberg** [comment]
In the "incorrect behavior" gif, are you unable to tab back into the about dialog at the end?

**Comment by lmulhall-phet** [comment]
> In fact, I might go so far as to say that there isn't actually a problem here. In the "incorrect behavior" I see that the close button is the first thing highlighted, and in the "correct behavior" I see that it is the last thing highlighted (JT is on old shas btw with that close button). Both seem like you can successfully navigate through the items in the about dialog.

OK, perhaps "incorrect" is too strong a word. However, I would like to point out that the behavior is inconsistent with all of the sims here https://phet.colorado.edu/en/accessibility/prototypes, including the published version of friction and friction 1.5.0-dev.5.

I assume the behavior in the published versions of sims with keyboard navigation is the _intended_ behavior, and I assume the behavior I'm seeing for this issue is _unintended_.

TL;DR: The behavior might not be "incorrect" but it is inconsistent.

**Comment by lmulhall-phet** [comment]
> In the "incorrect behavior" gif, are you unable to tab back into the about dialog at the end?

Yes. There's no loss of functionality with this behavior. I only pointed it out because it's inconsistent with the behavior of other simulations.

**Comment by zepumph** [comment]
I agree @lmulhall-phet, the about dialog's keyboard order is different that it has been in the past. Things are evolving. @jessegreenberg is the current behaviour (close button first, then links) correct? Or should it be instead implemented to go to the links first. If the current behavior seems alright. Please close this issue.

@lmulhall-phet as always, if it seems like I am not understanding, please step in.

**Comment by jessegreenberg** [comment]
Thanks for bringing this up @lmulhall-phet, I see now that the behavior is different and generally that is an indication of a new bug.  But in this case, the current behavior is correct and it was changed because it was odd for focus to automatically move to the last item in the dialog when opened. Closing.
